test/other/embind_tsgen_ignore_2.d.ts

```diff
@@ -105,8 +105,10 @@ interface EmbindModule {
   smart_ptr_function(_0: ClassWithSmartPtrConstructor): number;
   smart_ptr_function_with_params(foo: ClassWithSmartPtrConstructor): number;
   function_with_callback_param(_0: (message: string) => void): number;
-  string_test(_0: ArrayBuffer|Uint8Array|Uint8ClampedArray|Int8Array|string): string;
+  string_test(_0: EmbindString): string;
   wstring_test(_0: string): string;
 }
+type EmbindString = ArrayBuffer|Uint8Array|Uint8ClampedArray|Int8Array|string;
```

Comment:
Shouldn't this type come above its usage in the file?

Reply:
I see, but I think it would still be more readable if we ordered the file like that.   Not a huge deal I guess.

Reply:
Moved it to the top.